tools/run_tests/run_tests.py

```diff
@@ -155,6 +155,8 @@ def configure(self, config, args):
                                                                        self.args.compiler)
 
   def test_specs(self):
+    if self.args.compiler == 'freepass' and self.args.travis:
+      return []
```

Comment:
In nearly every other code review these days I say "never write a list where a tuple will do".

This file might just have to wait, it might look weird to be inconsistent and it's much too much of a change to make in the same pull request as something semantically unrelated.